schema/method_units/mathematical/diff.json

```diff
@@ -0,0 +1,20 @@
+{
+    "schemaId": "method-unit-diff",
```

Comment:
1. Should we keep the `schemaId` to be equal to the path with `/` replaced with `-`?
2. Since the content of the `method` directory is relatively light, we could consider nesting the `category` and `directory` folders under it

Reply:
The use of the underscores in names makes the above scheme hard to apply...

Reply:
Yes, the implicit conversion of an underscore to a hyphen (see also `core/reusable/atomic_vectors.json`) for the schema ID can cause issues, if, for instance, the schema ID is interpreted as a file path. Whatever the final strategy, it should be applied to all schemas. Having said that, I think `/` as a separator is a good idea, as no implicit conversion has to be considered.

Reply:
Per our earlier discussion, we'll deal with this later in a separate task.